public/index.html

```diff
@@ -8,11 +8,13 @@
     <meta name="description" content="DK Dojo | Personal Trainer"/>
 
     <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-4QVLR4G9QE"></script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-4QVLR4G9QE%22%3E"></script>
```

Comment:
SORRY, my bad... Can we get rid of the `%22%3E`?  This is the same for the other gtag issue.

Reply:
Ok. Removed it.